install.sh

```diff
@@ -971,4 +731,580 @@ azk8spull() {
 	fi
 }
 
+gen_crd_manifests() {
+    local crd=$1
+    need_cmd curl
+    ensure curl -sSL $crd
+}
+
+gen_chaos_mesh_manifests() {
+    local runc=$1
+    local socketPath="/var/run/docker.sock"
+    local mountPath="/var/run/docker.sock"
+    if [ "${runc}" == "containerd" ]; then
+        socketPath="/run/containerd/containerd.sock"
+        mountPath="/run/containerd/containerd.sock"
+    fi
+
+    need_cmd mktemp
+    need_cmd openssl
+    need_cmd curl
+
+    K8S_SERVICE="chaos-mesh-controller-manager"
+    K8S_NAMESPACE="chaos-testing"
+    tmpdir=$(mktemp -d)
+
+    ensure openssl genrsa -out ${tmpdir}/ca.key 2048 > /dev/null 2>&1
+    ensure openssl req -x509 -new -nodes -key ${tmpdir}/ca.key -subj "/CN=${K8S_SERVICE}.${K8S_NAMESPACE}.svc" -days 1875 -out ${tmpdir}/ca.crt > /dev/null 2>&1
+    ensure openssl genrsa -out ${tmpdir}/server.key 2048 > /dev/null 2>&1
+
+    cat <<EOF > ${tmpdir}/csr.conf
+[req]
+prompt = no
+req_extensions = v3_req
+distinguished_name = dn
+[dn]
+CN = ${K8S_SERVICE}.${K8S_NAMESPACE}.svc
+[v3_req]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = serverAuth
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = ${K8S_SERVICE}
+DNS.2 = ${K8S_SERVICE}.${K8S_NAMESPACE}
+DNS.3 = ${K8S_SERVICE}.${K8S_NAMESPACE}.svc
+EOF
+
+    ensure openssl req -new -key ${tmpdir}/server.key -out ${tmpdir}/server.csr -config ${tmpdir}/csr.conf > /dev/null 2>&1
+    ensure openssl x509 -req -in ${tmpdir}/server.csr -CA ${tmpdir}/ca.crt -CAkey ${tmpdir}/ca.key -CAcreateserial -out ${tmpdir}/server.crt -days 1875 -extensions v3_req -extfile ${tmpdir}/csr.conf > /dev/null 2>&1
+
+    TLS_KEY=$(openssl base64 -A -in ${tmpdir}/server.key)
+    TLS_CRT=$(openssl base64 -A -in ${tmpdir}/server.crt)
+    CA_BUNDLE=$(openssl base64 -A -in ${tmpdir}/ca.crt)
+
+    cat <<EOF
+---
```

Comment:
Is it possible to not copy all these yaml templates in this script? It would be a pain that we have to update both this script and the helm templates when there is a change.

Reply:
agree

Reply:
I just want to provide a more convenient installation method and do not rely on helm, Do you have any other good solutions?

Reply:
Maybe we can add a hack script to help use us update this.

Reply:
Maybe a hack script can be executed  after `make yaml` and checked by CI

Reply:
Done